ci/run.sh

```diff
@@ -21,7 +21,7 @@ fi
 if [[ "$(uname -s)" == 'Linux' ]]; then
     source conan/bin/activate
 else
-    export CMAKE_OPTIONS="$CMAKE_OPTIONS -DEXIV2_ENABLE_NLS=OFF"
+    export CMAKE_OPTIONS="$CMAKE_OPTIONS -DCMAKE_PREFIX_PATH=/usr/local/opt/gettext/"
```

Comment:
Is manually adding the prefix necessary? (This is not a nitpick, I have no clue about OSX, I would have assumed that cmake would pick this up)

Reply:
Yes, sometimes it is needed to do this kind of things. For example when working with Qt, it is also needed to help CMake to find external libraries. I think this is fine.

Reply:
Yeah, this is more of a Homebrew issue than an OSX thing. They put certain packages in non-standard places so that they don't conflict with system-provided things. It makes Mac development unwieldy sometimes.